config/sync/user.role.site_admin.yml

```diff
@@ -8,6 +8,7 @@ dependencies:
     - taxonomy.vocabulary.uswds_breakpoints
     - taxonomy.vocabulary.uswds_classes
   module:
+    - action
```

Comment:
Aha! This change (adding `-action` in user.role.site_admin.yml) needs to go. It's probably the result of a bad merge or something like that. 
Removal of the `action` module (Actions UI) has already been merged to dev, although it was not in the database snapshot I was working with, and this caused `drush cim` to fail with
```
[error]  Drupal\Core\Config\ConfigImporterException: There were errors validating the config synchronization.
Configuration <em class="placeholder">user.role.site_admin</em> depends on the <em class="placeholder">Actions UI</em> module that will not be installed after import. in Drupal\Core\Config\ConfigImporter->validate() (line 823 of /var/www/web/core/lib/Drupal/Core/Config/ConfigImporter.php). 
```